docs/learn/learn-nominator.md

```diff
@@ -244,11 +253,14 @@ each.
   2) Total amount of nominators that nominated that validator.
 
   - You may want to be cautious of validators with a high number of subscribers. A validator is
-    considered oversubscribed when more than 256 'active' nominators are assigned to the validator. In this
-    scenario only the top 256 nominators will receive rewards. The remaining nominators will
+    considered oversubscribed when more than 
+    {{ polkadot: <RPC network="polkadot" path="query.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="query.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :kusama }}  'active' nominators are assigned to the validator. In this
+    scenario only the top {{ polkadot: <RPC network="polkadot" path="query.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="query.staking.maxNominatorRewardedPerValidator" defaultValue={256}/> :kusama }} nominators will receive rewards. The remaining nominators will
     recieve nothing, however they can be slashed in the event that validator commits a
     slashable offence.
-  - Every nominator can select up to a maxium of 16 validators, which contributes towards maximizing
+  - Every nominator can select up to a maxium of {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominations" defaultValue={16}/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }} validators, which contributes towards maximizing
```

Comment:
Everything looks good to me, I would just suggest new-lining these few components to keep the consistency.  Otherwise running prettier may cause issues.